gateway/metrics/exporter.go

```diff
@@ -72,9 +82,45 @@ func (e *Exporter) StartServiceWatcher(endpointURL url.URL, metricsOptions Metri
 	ticker := time.NewTicker(interval)
 	quit := make(chan struct{})
 
-	timeout := 3 * time.Second
+	go func() {
+		for {
```

Comment:
What is this new segment of code for? I was mainly expecting this PR to set a default suffix in the invocation URL when there was not one given.

Reply:
New segment of code is for getting replica count of the services, which was not having support for multiple namespace. It was only getting replica count for the functions present in default namespace.